### ISSUE TITLE
[BUG] Axes Labels overlapping

### ISSUE THREAD

**Issue by loretoparisi** [plotly/dash-core-components#588]
**Describe your context**
We have axis label texts overlapping. We do no use any custom CSS.

-  replace the result of `pip list | grep dash` below
```
dash (0.43.0)
dash-core-components (0.42.1)
dash-daq (0.1.5)
dash-html-components (0.13.4)
dash-renderer (0.24.0)
dash-table (3.1.11)
```
-  if frontend related, tell us your Browser, Version and OS

       macOS 10.14.5 (18F132)
      Chrome
       74.0.3729.169

**Describe the bug**

**Expected behavior**

**Screenshots**

Here we have a label placed on top.
<img width="1435" alt="Schermata 2019-06-11 alle 11 24 50" src="https://user-images.githubusercontent.com/163333/59260254-9149c200-8c3b-11e9-8062-0c59ba9d0f00.png">

Here we have a label placed on the bottom.
<img width="1455" alt="Schermata 2019-06-11 alle 11 24 36" src="https://user-images.githubusercontent.com/163333/59260263-960e7600-8c3b-11e9-82ae-0e507770fdf5.png">

**Comment by loretoparisi** [comment]
@byronz any update on this issue? I have `dash==1.6.0` and it's still there. Thanks

**Comment by byronz** [comment]
@shammamah  can you take a look if you have time ? thanks

**Comment by shammamah** [comment]
@loretoparisi Would you be able to provide a minimal working example that replicates this issue?

**Comment by loretoparisi** [comment]
@shammamah yes, we will proved it, thank you for your help cc @shoegazerstella

**Comment by ntentes** [comment]
Any update on this?
